### ISSUE TITLE
Add new plugin for Sort

### ISSUE THREAD

**Issue by zuketo** [ethereum/remix-project#2885]
Hello, I believe I need to make a PR against a specific file for submitting a new plugin, can you please direct me to the file and I'll create the PR against this issue.

The plugin is for sort.xyz

Here is the Plugin repo: https://github.com/sortxyz/sort-remix-plugin

And profile.js: https://github.com/sortxyz/sort-remix-plugin/blob/master/resources/profile.js

The plugin is live at remix.sort.xyz

Thanks!
Jason

**Comment by yann300** [comment]
Thanks for the message,
could you send us a video demo of this plugin?

**Comment by zuketo** [comment]
Hi @yann300 I created a video overview here: [https://www.youtube.com/watch?v=v4rM42eT6NM](https://www.youtube.com/watch?v=v4rM42eT6NM)

Please let me know if I can follow up on any part of it.

Thanks!

**Comment by zuketo** [comment]
Hi @yann300 I was curious if you had a chance to review?

**Comment by yann300** [comment]
Yes thanks fort the heads up I will spend some time today to check it.
Might also have be nice to set it as main panel plugin, so there'd be enough place for the whole app.

**Comment by yann300** [comment]
I ran `npm run build` but it doesn't produce the `index.html`, can you help with this?
thanks
